projects/mono.md

```diff
@@ -0,0 +1,31 @@
+# Mono
+
+Mono, the open source development platform based on the .NET Framework, allows developers to build cross-platform applications with improved developer productivity. Mono’s .NET implementation is based on the ECMA standards for [C#](http://www.ecma-international.org/publications/standards/Ecma-334.htm) and the [Common Language Infrastructure](http://www.ecma-international.org/publications/standards/Ecma-335.htm).
+
+Supported previously by Novell, [Xamarin](http://xamarin.com/) and now [Microsoft](http://www.microsoft.com/) and the [.NET Foundation](https://dotnetfoundation.org/), the Mono project has an active and enthusiastic contributing community. Mono includes both developer tools and the infrastructure needed to run .NET client and server applications.
+
+## The Components
+
+There are several components that make up Mono:
+
+* **C# Compiler** - Mono’s C# compiler is feature complete for C# 1.0, 2.0, 3.0, 4.0 and 5.0 (ECMA). A good description of the feature of the various versions is available on [Wikipedia](http://en.wikipedia.org/wiki/C_Sharp_%28programming_language%29#Versions).
```

Comment:
Again, there is no ECMA specification for C# 3.0 - 5.0. And [on the Mono site](http://www.mono-project.com/docs/about-mono/languages/csharp/), where this was probably copied from, the "(ECMA)" makes at least some sense, since it's a link.